pwndbg/aglib/dynamic.py

```diff
@@ -68,7 +58,7 @@ def is_dynamic() -> bool:
 R_DEBUG_LINK_MAP_CHANGED_LISTENERS: Set[Callable[..., Any]] = set()
 
 
-class RDebugLinkMapChangedHook(pwndbg.gdblib.bpoint.BreakpointEvent):
+def r_debug_link_map_changed_hook() -> Callable[[pwndbg.dbg_mod.StopPoint], bool]:
```

Comment:
Honestly, I prefer BreakpointEvent from a StopPoint but w/e for now

Reply:
The name `StopPoint` in the Debugger-agnostic API comes from the fact that it's generic over both breakpoints and watchpoints. We could just use the name `Breakpoint` for both, though I do lean towards not overloading terms, but ultimately this is something that can be changed fairly easily